### PR TITLE
Removing space

### PR DESCRIPTION
**Pull request by SOUMOJIT-CHAKRABORTY**
I have remove a space

### CODE REVIEW

**Comment by Windvis** [comment]
But... why? :smile:

**Comment by SOUMOJIT-CHAKRABORTY** [comment]
> But... why? 😄

Actually I am new to opensource, I have just learnt javascript and don't know how to contribute in actual project. I tried to reach out people but they did't reply back. I don't know anyone how can guide me, so I tried to contribute in this way.😞

**Comment by Windvis** [comment]
@SOUMOJIT-CHAKRABORTY I would suggest that you [join the discord](https://discord.gg/emberjs) if you haven't done so already. There are a lot of very friendly people there. Most people receive a lot of Github notifications so they might not have seen your message. The Discord will probably yield better results.

You could take a look at the help-wanted website: https://help-wanted.emberjs.com/ It shows an overview of ember-repositories that are looking for help. I would go through those until you find something that interests you.

Contributing to addons is also an option. There are a lot of addons that would love the help, for example the [`adopted-ember-addons` project](https://github.com/adopted-ember-addons) has a lot of addonss that are very popular (and which might not have a maintainer).

Long story short, there is a lot that you can do, but opening pull requests with seemingly random contributions isn't the most productive :smile:. Just try to find something that interests you, and don't hesitate to ask questions in the Discord.

**Comment by SOUMOJIT-CHAKRABORTY** [comment]
Thank you sir for you guidance, I have join the discord community. Sir I need one more help, can you tell me how much javascript should I learn to start my opensource journey? It will be helpful.🙂
